core/domain/stats_jobs_one_off_test.py

```diff
@@ -29,6 +32,81 @@
     [models.NAMES.exploration, models.NAMES.statistics])
 
 
+class RemoveBadPlaythroughsOneOffJobTest(test_utils.GenericTestBase):
+    exp_id1 = 'exp_id1'
+
+    def setUp(self):
+        super(RemoveBadPlaythroughsOneOffJobTest, self).setUp()
+        config_services.set_property(
+            'committer_id1', 'whitelisted_exploration_ids_for_playthroughs',
+            [self.exp_id1])
+        self.model_id = (
+            stats_models.PlaythroughModel.create(
+                self.exp_id1, 1, 'EarlyQuit', {}, []))
+        playthrough_instance = stats_models.PlaythroughModel.get(self.model_id)
+        playthrough_instance.created_on = datetime.datetime(
+            2018, 9, 1, 7, 47, 0, 0)
+        playthrough_instance.put()
+        stats_models.ExplorationIssuesModel.create(
+            self.exp_id1, 1,
+            [{
+                'issue_type': 'EarlyQuit',
+                'issue_customization_args': {
+                    'state_name': {
+                        'value': 'state_name1'
+                    },
+                    'time_spent_in_exp_in_msecs': {
+                        'value': 200
+                    }
+                },
+                'playthrough_ids': [self.model_id],
+                'schema_version': 1,
+                'is_valid': True
+            }]
+        )
+
+    def test_playthrough_is_deleted(self):
```

Comment:
One more case to test: issue contains multiple playthroughs, but only some of them are affected.

Reply:
Done.